agent/util-scripts/pbench-tool-meister-start

```diff
@@ -353,6 +356,17 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    try:
+        inst_dir = PbenchAgentConfig(os.environ["_PBENCH_AGENT_CONFIG"]).pbench_install_dir
+        tm_start_path = Path(inst_dir).resolve(strict=True)
+    except FileNotFoundError:
+        logger.error("Unable to determine proper installation directory, {inst_dir} not found")
```

Comment:
Here I think you need to `return 1` or some error number value to indicate the `pbench-tool-meister-start` command was not successful.

Reply:
One more point, when using the python logging sub-system, we typically don't want to fully format strings ahead of time.  The logging sub-system gives us a way to do that at run time to avoid the formatting if the log message is not going to be emitted.

In this case it is not a big deal because we are typically always going to emit the message, but the discipline is worth maintaining to avoid making that mistake with `.info` or `.debug` log statements.

Consider: `logger.error("Unable to determine proper installation directory, '%s' not found", inst_dir)`

Reply:
Further, I am not entirely sure, but the code for `PbenchConfig` and `PbenchAgentConfig` classes don't appear to explicitly raise a `FileNotFoundError`, but neither to they prevent that error from being raised for their operation.

It might be better to consider handling the exception separately, something like:

```
try:
    inst_dir = PbenchAgentConfig(...).pbench_install_dir
except BadConfig as exc:
    logger.error("%s", exc)
    return 1
except Exception:
    logger.error("Unexpected error encountered logging pbench agent configuration", exc)
    return 1

try:
    tm_start_path = Path(inst_dir).resolve(strict=True)
except ...
```